presets/ragengine/vector_store/base.py

```diff
@@ -281,13 +282,50 @@ async def _persist_internal(self, index_name: str, path: str):
             if index_name not in self.index_map:
                 raise HTTPException(status_code=404, detail=f"No such index: '{index_name}' exists.")
 
-            logger.info(f"Persisting index {index_name} into {path}.")
-            await asyncio.to_thread(self.index_store.persist, os.path.join(path, "store.json"))
-
```

Comment:
This `store.json` was already getting saved by `storage_context.persist(path)` it was getting double saved, so I removed it.